src/io/windows.zig

```diff
@@ -1394,8 +1394,19 @@ pub const IO = struct {
             };
         }
 
+        // Declaring the function with an alternative name because `CamelCase` functions are
+        // by convention, used for building generic types.
```

Comment:
TBH, I'd remove the comment --- it is not particularly relevant to what we are doing here so it distracts the reader. If anyone tries to rename it to be more idiomatic, they'll learn why it is the way it is :)

Reply:
But that would be great tidbit to include in the commit message: https://mislav.net/2014/02/hidden-documentation/